bin/cython_freeze

```diff
@@ -51,6 +64,23 @@ print("""\
 for name in modules:
     print("PyMODINIT_FUNC MODINIT(%s) (void);" % name)
 
+for package in packages:
+    print('''
+    
+    static PyModuleDef %sModule = {
+        PyModuleDef_HEAD_INIT, "%s", NULL, -1, NULL,
+        NULL, NULL, NULL, NULL
+        };
+
+    static PyObject * PyInit_%s(void)
+    {
+        PyObject * mod = PyModule_Create( & %sModule);
+        PyModule_AddObject(mod, "__path__", Py_BuildValue("()"));
+        return mod;
```

Comment:
The generated C code lacks error handling everywhere. Could you please add that?

Reply:
Changes were made.